helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/bootstrap/HelpDeskBootstrap.java

```diff
@@ -73,6 +75,7 @@ public void execute(ApplicationContext applicationContext) {
         .onError(e -> LOGGER.error("Fail to initilize Helpdesk Ai", e));
 
     try {
+      executeProvisioning(applicationContext);
```

Comment:
You should execute this using FunctionExecutor to ensure retry policy will be applied.

Reply:
Done.